game.rb

```diff
@@ -24,21 +23,28 @@ def player_one_pick_word
   end
 
   def player_two_guess_word word
-    counter = 0
+    guess_counter = 0
     end_of_game = false
+    @io_handler.print(@game_view.body_array)
     while ! end_of_game
```

Comment:
This while loop is getting pretty huge. It would be worth breaking it down into methods that represent each part.